SpatialGDK/Source/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -3,6 +3,7 @@
 #include "SpatialWorkerConnection.h"
 
 #include "Async/Async.h"
+#include <string>
```

Comment:
Don't need this any more. Sorry, added it in an experiment and forgot to remove it.